.github/workflows/build.yml

```diff
@@ -9,6 +9,8 @@ env:
   DISABLE_KNAPSACK: true
   TIMEZONE: UTC
   COVERAGE: true
+  STRIPE_INSTANCE_SECRET_KEY: 'sk_test_9x7Au91q3BRmsPi1H2l6qvIf00lpLDKToD'
+  STRIPE_INSTANCE_PUBLISHABLE_KEY: 'pk_test_eEobkSV1pzlSnnXIxf1vXvOm00ASnsfCDd'
```

Comment:
yeah, the No API key errors are not happening locally for me.
For some reason these are not working in github:
allow(Stripe).to receive(:api_key) { "sk_test_xxxx" }


But do you understand why would we need stripe test keys if we are just using stubs? Have you tried to define these  env variables but with dummy values?
STRIPE_INSTANCE_SECRET_KEY: 'sk_test_xxxx')
Maybe that will also work?

Reply:
ok, I am giving it a try here:
https://github.com/openfoodfoundation/openfoodnetwork/pull/7554

Another option is to just set Stripe.api_key = "sk_test_xxxx" in the specs instead of using allow.

Reply:
yes, it works.
This env var on the build is acceptable, but I think we will be able to replace
allow(Stripe).to receive(:api_key) { "sk_test_xxxx" }
with
Stripe.api_key = "sk_test_xxxx"

Reply:
You are correct! I pushed that suggestion to your rails 6 branch:

https://github.com/openfoodfoundation/openfoodnetwork/runs/2535547307

I'm going to close this PR and submit a new one.